nss-evening-cohort-24/pos-system-the-ronis
# Authentication Page

## Description
Whenever the page is opened, there should be a logo as well as a button to log in. Once the log in button is selected, a request for authentication should appear. Once the user authenticates, the navbar as well as the rest of the POS application loads. 

## Related Issue
#7 

## Motivation and Context
This page is essential to the presentation of the site. It serves as a gateway. 

## How Can This Be Tested?
- Pull PR branch to local machine, run server, and make sure logo renders, login button can be selected, and authentication is requested once login page loads. Once authenticated, you should see the rest of the site. 

## Screenshots (if appropriate):
![image](https://github.com/nss-evening-cohort-24/pos-system-the-ronis/assets/97858910/d0e8cc44-18f1-4fcd-8398-48edade2be66)

## Types of changes
